app/helper/db/rt_plugin_info.php

```diff
@@ -1,63 +1,131 @@
 <?php
-
 /**
- * Description of rt_plugin_info
+ * Class to store rtMedia plugin info.
+ *
+ * @package    rtMedia
  *
  * @author udit
  */
 
 if ( ! class_exists( 'rt_plugin_info' ) ) {
+	/**
+	 * Class to store rtMedia plugin info.
+	 */
 	class rt_plugin_info {
 
-		//put your code here
+		/**
+		 * Plugin path.
+		 *
+		 * @var string $plugin_path
+		 */
 		public $plugin_path;
-		public $name; //'Name' - Name of the plugin, must be unique.
-		public $title; //'Title' - Title of the plugin and the link to the plugin's web site.
-		public $desctipriton; //'Description' - Description of what the plugin does and/or notes from the author.
-		public $authro; //'Author' - The author's name
-		public $authoruri; //'AuthorURI' - The authors web site address.
-		public $version; //'Version' - The plugin version number.
-		public $pluginuri; //'PluginURI' - Plugin web site address.
-		public $textdomain; //'TextDomain' - Plugin's text domain for localization.
-		public $domain_path; //'DomainPath' - Plugin's relative directory path to .mo files.
-		public $network; //'Network' - Boolean. Whether the plugin can only be activated network wide.
+
+		/**
+		 * 'Name' - Name of the plugin, must be unique.
+		 *
+		 * @var string $name
+		 */
+		public $name;
+
+		/**
+		 * 'Title' - Title of the plugin and the link to the plugin's web site.
+		 *
+		 * @var string $title
+		 */
+		public $title;
+
+		/**
+		 * 'Description' - Description of what the plugin does and/or notes from the author.
+		 *
+		 * @var string $desctipriton
+		 */
+		public $desctipriton; // TODO : Correct spelling error.
```

Comment:
Why you have added todo instead of renaming variable?

Reply:
Have to check in other add-ons if they using this class as base. So other class might be using this variable.

Reply:
Okay. Please create an issue to track this issue.